.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "editor.formatOnSave": true
```

Comment:
So I don't think we want this in the repo at all, looks like your local IDE configuration. Again .gitignore should be cutting this out from being added to the commit at all. I'll make a quick update to the master .gitignore and then we can try merge this and see if it cuts it out.